### PR TITLE
Address user story #5

### PR DESCRIPTION
**Pull request by ojacques**
Add 25% discount

### CODE REVIEW

**Comment by D4rkPa5senger** [comment]
It should be 50%, 25% is not acceptable! Change it to 50%...

File: monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
-GLOBAL_DISCOUNT=0
+GLOBAL_DISCOUNT=25
```
Comment (ojacques) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: @D4rkPa5senger - per user story #5 , marketing told us to apply 25% discount. Do you agree? ❓ 😏

Comment (D4rkPa5senger) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: I will talk top marketing, don't worry about that! Just make it 50%

Comment (Brockdi) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Parth, you will make our company go broke!

Comment (infrautocode) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Agreed as @D4rkPa5senger commented.

Comment (Rossdar) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: I vote for 50%.  We need to get sales moving...

Comment (Brockdi) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Make it 100%!

Comment (ActiveD5) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Can't wait to jump on this huge discount!

Comment (JeanBennett01) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Ever hear of CALMS?

**Comment by ojacques** [comment]
Cats are awesome 🐈 😸

File: monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties (hunk 1 above)
Comment (Rossdar) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: 50% looks good!!

Comment (ActiveD5) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: `Who cares what marketing says?  `

Comment (JeanBennett01) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: We could just pay the customer to buy!

File: monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties [hunk 2]
```diff
@@ -1,2 +1,2 @@
-GLOBAL_DISCOUNT=50
+GLOBAL_DISCOUNT=33
```
Comment (D4rkPa5senger) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: I thought I told you to make it 50% :angry: :rage:

Comment (Brockdi) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: you forgot the 1 before 33

Comment (ojacques) [inline monolithic/common/util/src/main/java/io/github/zutherb/appstash/common/util/config.properties]: Ooops 😨 😨

**Pull request merged**